Source/Diagnostics/ReducedDiags/FieldEnergy.cpp

```diff
@@ -37,7 +37,7 @@ FieldEnergy::FieldEnergy (std::string rd_name)
 
     constexpr int noutputs = 3; // total energy, E-field energy and B-field energy
     // resize data array
-    m_data.resize(noutputs*nLevel,0.0);
+    m_data.resize(noutputs*nLevel, amrex::Real(0.0));
```

Comment:
You should be able to do this, if you want to be more compact
```suggestion
    m_data.resize(noutputs*nLevel, 0.0_rt);
```

Reply:
In addition, lgtm complains about the product `noutputs*nLevel`, stating that it could overflow `int` before being converted to `size_t` (well, it's quite unlikely ;-) ). I think that a `static_cast<size_t>` should be enough to silence this warning.